test/v2/controllers/V2DeparturesControllerSpec.scala

```diff
@@ -787,13 +785,13 @@ class V2DeparturesControllerSpec
     }
   }
 
-  def generateResponse(messageId: MessageId) =
-    MessageSummary(
-      messageId,
-      arbitrary[OffsetDateTime].sample.value,
-      DepartureDeclaration,
-      Some("<CC015C><test>testxml</test></CC015C>")
-    )
+//  def generateResponse(messageId: MessageId) =
+//    MessageSummary(
+//      messageId,
+//      arbitrary[OffsetDateTime].sample.value,
+//      DepartureDeclaration,
+//      Some("<CC015C><test>testxml</test></CC015C>")
+//    )
```

Comment:
We don't leave commented code unless there are exceptional reasons.

Reply:
Yes. We don't - I'll remove it.